base64/base64.rs/src/bin/base64.rs

```diff
@@ -10,17 +9,27 @@ const TRIES: usize = 8192;
 fn main() {
     let base64_engine = base64::engine::general_purpose::STANDARD;
 
-    for [src, dst] in &[["hello", "aGVsbG8="], ["world", "d29ybGQ="]] {
-        let encoded = base64_engine.encode(src);
+    for (src, dst) in [(b"hello", b"aGVsbG8="), (b"world", b"d29ybGQ=")] {
+        let mut encoded = [0u8; 8];
+        base64_engine.encode_slice(src, &mut encoded).unwrap();
```

Comment:
It's fine to extend the tests, but the original idea is testing the API used in the benchmark as we don't have the visibility into the hot path execution (we already had the situation when API didn't work correctly). Please add (or revert) the testing of `encode`/`decode` functions used in the benchmark.